processor/attributesprocessor/testdata/config.yaml

```diff
@@ -102,7 +105,9 @@ processors:
   attributes/includeservices:
     # Specifies the span properties that must exist for the processor to be applied.
     include:
-      # The Span service name must either be "svcA" or "svcB" to match.
+      # match_type defines that "services" is an array of regexp-es.
+      match_type: regexp
+      # The Span service name must match "svcA" or "svcB" regexp.
```

Comment:
Not sure what matching these strings means, it is not a strict :). Maybe better examples?

Reply:
Yes, let me change the example.

Reply:
Fixed.